Readme.md

```diff
@@ -31,6 +31,7 @@ Your user also needs permission to access `/dev/uhid` so that `tpm-fido` can app
 I use the following udev rule to set the appropriate `uhid` permissions:
 
 ```
+KERNEL=="tpmrm[0-9]*", MODE="066", GROUP="SOME_UHID_GROUP_MY_USER_BELONGS_TO"
```

Comment:
This conflicts with the above instructions to add your user to the tss group. Does Suse set different permissions than this for `tpmrm`:

```$ ls -l /dev/tpmrm0 
crw-rw---- 1 tss tss 253, 65536 Sep 16 17:13 /dev/tpmrm0
```

Reply:
Per default on my system, only `/dev/tpm0` is owned by `tpm` but has group `root`.
`/dev/tpmrm0` is owned by `root` and group `root` with permission `600` be default.